scripts/add_electricity.py

```diff
@@ -240,6 +240,16 @@ def load_costs(tech_costs, config, max_hours, Nyears=1.0):
     costs.loc[costs.unit.str.contains("/GW"), "value"] /= 1e3
     costs.unit = costs.unit.str.replace("/GW", "/MW")
 
+    for attr in ("investment", "lifetime", "FOM", "VOM", "efficiency", "fuel"):
+        overwrites = config.get(attr)
+        if overwrites is not None:
+            breakpoint()
+            overwrites = pd.Series(overwrites)
+            costs.loc[overwrites.index, attr] = overwrites
+            logger.info(
+                f"Overwriting {attr} of {overwrites.index} to {overwrites.values}"
+            )
```

Comment:
`logger` is not implemented everywhere. It may be useful to add it each time we override values.

Reply:
Good point! Maybe it makes sense also to add that an adjusted cost file `costs_*_adjusted\.csv` is saved? What do you think?

Reply:
Would you rather check what was used in a file or in the network itself? I would probably trust the network more.